### ISSUE TITLE
Js cookie cannot handle expiration times less than 1 day

### ISSUE THREAD

**Issue by endymon** [js-cookie/js-cookie#74]
Is there a reason why js cookie only handles cookie expiration times in increments of days?
It seems like it would make a lot of sense to have a couple of designiated units of time: year, month, day, hour, minutes, seconds.
Which one is the default doesn't really matter (though for backward compat could use day).

Before I go about adding that functionality myself I was wondering if there was a specific reason it does not exist already.

Thanks

**Comment by carhartl** [comment]
Back when I wrote the jQuery cookie plugin days seemed the easiest/most usable (and most used) default for expiration dates. Seeing the need for more flexibility, soonafter the support for using `Date` instances were added. With this, all needs are covered, in addition we later added the ability to specify a fraction of a day, so, say you want the cookie to expire in 1 hour, you'd specify: `expires: 1/24` (not too intuitive, I know, but it was expected by users to work like this).

What you're asking for on the other hand is basically some sort of time unit parser, which would add much complexity requiring documentation, testing, and would add to the file size as well, which we're trying to keep as small as possible. But more importantly, it can be and is already covered by plain old JavaScript (`Date`) avoiding the need for users to lookup documentation and remember the various options.

To sum it up: Use `Date` :)
